packages/devtools_app/lib/src/banner_messages.dart

```diff
@@ -234,26 +234,33 @@ class DebugModePerformanceMessage {
     return _BannerError(
       key: Key('DebugModePerformanceMessage - $screenId'),
       textSpans: [
-        const TextSpan(
+        TextSpan(
           text: '''
 You are running your app in debug mode. Debug mode performance is not indicative of release performance.
 Relaunch your application with the '--profile' argument, or ''',
-          style: TextStyle(color: _BannerError.foreground),
+          style: TextStyle(
+            color: _BannerError.foreground,
+            fontSize: defaultFontSize,
```

Comment:
we failed to use our default font size every time we emitted a custom TextStyle without a font size.

Reply:
can we add defaultFontSize to the theme so that any `TextStyle` without a `fontSize` specified gets `defaultFontSize` as its `fontSize`?

Reply:
Hmm, I think the only way to do that is MediaQueryData.textScaleFactor
which might be a lot cleaner than what we are currently doing but we would need to be careful not to play badly with accessibility tools.

Reply:
Tested and it didn't fix this case and caused other problems so sticking with this approach.